### PR TITLE
ADAPT-6374 BEM and REM changes, indentation updates, boolean updates.…

### PR DESCRIPTION
**Pull request by ethan-lp**
… Issue with image and text displaying and 180 flip, text should be on back of image

### CODE REVIEW

**Review by eleanor-heath — APPROVED** [review]

File: js/adapt-contrib-flipcard.js [hunk 1]
```diff
@@ -1,214 +1,215 @@
...
-            }
+  'coreViews/componentView',
+  'coreJS/adapt',
+  'underscore'
+], function(ComponentView, Adapt){
+
+  class Flipcard extends ComponentView {
+    events() {
+      return {
+      'click .flipcard__item': 'onClickFlipItem'
```
Comment (cahirodoherty-learningpool) [inline js/adapt-contrib-flipcard.js]: Minor: indentation looks off here

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/adapt-contrib-flipcard.js (hunk 1 above)
Comment (ethan-lp) [reply js/adapt-contrib-flipcard.js]: Sorted

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/adapt-contrib-flipcard.js [hunk 2]
```diff
@@ -104,87 +88,86 @@ define([
...
-      var hasBeenFlipped = $selectedElement.hasClass('flipcard__flip');
-      var $front = $selectedElement.find('.flipcard__item-front');
-      var $back = $selectedElement.find('.flipcard__item-back');
-      var $textElements = $back.children();
-
-      Adapt.a11y.toggleAccessibleEnabled($textElements, hasBeenFlipped); 
-      Adapt.a11y.toggleHidden($front, hasBeenFlipped);
-      Adapt.a11y.toggleHidden($back, !hasBeenFlipped);
-    }
+    // This function is responsible for rotating an individual item
```
Comment (cahirodoherty-learningpool) [inline js/adapt-contrib-flipcard.js]: This should be deleted instead of commented out

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**